eng/pipelines/templates/steps/verify-versions.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+- task: PowerShell@2
```

Comment:
Why make a template for this as opposed to adding the one step in the other template?